client/src/utils/deleteTokenOn401StatusCodes.js

```diff
@@ -0,0 +1,19 @@
+export default (axios) => {
+  const successHandler = (response) => {
+    return response;
+  };
+
+  const errorHandler = (error) => {
+    if (error.response.status === 401) {
+      window.localStorage.removeItem('jwt_token');
+    }
+    return Promise.reject(error);
+  };
+
+  axios.interceptors.response.use(
```

Comment:
extra spice

Reply:
triple flame

Reply:
I don't understand this but it does indeed look spicy